cts_runner/src/main.rs

```diff
@@ -157,6 +155,9 @@ mod native {
         ops = [op_exit, op_read_file_sync, op_write_file_sync],
         esm_entry_point = "ext:cts_runner/src/bootstrap.js",
         esm = ["src/bootstrap.js"],
+        state = |state| {
+            state.put(Permissions {});
+        }
```

Comment:
permissions must be available in state before bootstrap. for cts_runner it's static, so i moved it here into the extension macro.